Content.Shared/Cluwne/CluwneBeastComponent.cs

```diff
@@ -0,0 +1,48 @@
+using Robust.Shared.Audio;
+using Content.Shared.Chat.Prototypes;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Shared.Cluwne;
+
+[RegisterComponent]
+[NetworkedComponent]
+public sealed class CluwneBeastComponent : Component
```

Comment:
should have access + you can merge the attributes with `[RegisterComponent, NetworkedComponent]`

Reply:
Access still needs doing.